js/actors/arm.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Player class. Contains all movement logic and instance variables.
+ */
+class Arm extends Actor {
+
+    constructor(player, x = 0, y = 300, w = 20 * 0.8, h = 36 * 0.8) {
```

Comment:
seems like we should be spawning relative to player position instead of an absolute position. I know it updates in act() but still